bulbs/liveblog/tasks.py

```diff
@@ -0,0 +1,16 @@
+from celery import shared_task
+import requests
+
+from django.conf import settings
+from django.utils import timezone
+
+
+@shared_task(default_retry_delay=5)
+def firebase_update_timestamp(liveblog_id):
+    endpoint = getattr(settings, 'LIVEBLOG_FIREBASE_NOTIFY_ENDPOINT', None)
+    if endpoint:
+        url = endpoint.format(liveblog_id=liveblog_id)
+        resp = requests.put(url, json={
+            'updatedAt': timezone.now().timestamp()  # TODO: What time format?
```

Comment:
If this is the same date format as the GET entries api:

`/api/v1/liveblog/public/entry/`

`if_modified_since=2015-01-01T01:01:00.000Z`

Then the javascript can drop the the `updatedAt` value into the url.

(Assuming the `if_modified_since` filter is inclusive.)